erpnext/hr/doctype/employee_loan/employee_loan.js

```diff
@@ -36,7 +36,21 @@ frappe.ui.form.on('Employee Loan', {
 		})
 	},
 
-	mode_of_payment: function(frm){
+	on_submit: function(frm) {
```

Comment:
Instead of creating JV on submit, give a button to make the disbursement/payment entry. Because, we don't have any control on whether user submitted the JV or not, after submitting the Loan record.